tests/TestHelpers/GraphHelper.php

```diff
@@ -446,6 +446,33 @@ public static function getGroups(
 		);
 	}
 
+	/**
+	 * @param string $baseUrl
+	 * @param string $groupId
+	 * @param string $xRequestId
+	 * @param string $user
+	 * @param string $password
+	 *
+	 * @return ResponseInterface
+	 * @throws GuzzleException
+	 */
+	public static function getSingleGroup(
```

Comment:
```suggestion
	public static function getGroup(
```
I think that should be clear that its a single one